### ISSUE TITLE
Unexpected unknown value "linear(0, 1)" for property "animation-timing-function"

### ISSUE THREAD

**Issue by in-in** [stylelint/stylelint#8379]
### What minimal example or steps are needed to reproduce the bug?

```css
a {
  transition-timing-function: linear(0 0%, 0.999 44.8%, 0.866 58.4%, 0.998 77.2%, 1 100%);
  animation-timing-function: linear(0, 1);
}

```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "declaration-property-value-no-unknown": true
  }
}
```

### How did you run Stylelint?

[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcICGACYAdAdugLjgTkpgM4CWOpEmAtBQLamYDm1AZgK6ZgVXwoA2jGEnwAKAAwpxAUgA0UgHQBOFSgAsahQA45irQDZ9KAKxaFa3eOVKtKAOx2FAJl0BGFK-EyAlAG4suESkdEg8NPSMLBxcYXyCmMJi4vKuflgAvlggsiCspPwwAHJIdHCIMAAeJQAOBQpgxMTZ4FR5TAggGJgoKOgg+OwFxH18XT09fbBg-CKhlDTV+BDVMPg4AJ7UAG5I-Oww1JgQ1JwA1kcA7tgIKAT7WD2ZmOnNkJhtAGIQ+CE4HQBWxCozVg1SaiDGvRAxA2BXiOBGUJmOBgML6sgeUJh6zhjBw1DebWoMKIUBEUERfWRqIRIAyIHSQA)

### Which Stylelint-related dependencies are you using?

```json
{
  "stylelint": "16.14.1",
  "stylelint-config-standard": "37.0.0"
}
```

### What did you expect to happen?

No problems to be reported.

### What actually happened?

```sh
2:30-89 error Unexpected unknown value "linear(0 0%, 0.999 44.8%, 0.866 58.4%, 0.998 77.2%, 1 100%)" for property "transition-timing-function" (declaration-property-value-no-unknown)
3:29-41 error Unexpected unknown value "linear(0, 1)" for property "animation-timing-function" (declaration-property-value-no-unknown)
```

### Do you have a proposal to fix the bug?

No

**Comment by jeddy3** [comment]
@in-in Thanks for the report and for using the template.

This will be fixed by https://github.com/stylelint/stylelint/issues/8100 as the [patch CSS syntax contain `linear()`](https://github.com/csstools/postcss-plugins/blob/4eefc5bd78bc4cf9e3a3a9d7f75039b04245f4c5/packages/css-syntax-patches-for-csstree/dist/index.json#L586).

https://github.com/stylelint/stylelint/issues/8100 is labelled as ready to implement. Please consider [contributing](https://stylelint.io/contributing) if you have time.

**Comment by skyclouds2001** [comment]
I've fired https://github.com/mdn/data/pull/951 to fix this issue in upstream
